files/build_templates/docker_image_ctl.j2

```diff
@@ -1,5 +1,31 @@
 #!/bin/bash
 
+# single instance containers are still supported (even though it might not look like it)
+# if no instance number is passed to this script, $DEV will simply be unset, resulting in docker 
+# commands being sent to the base container name. E.g. `docker start database$DEV` simply starts
+# the container `database` if no instance number is passed since `$DEV` is not defined
+
+
+{%- if docker_container_name == "database" %}
+link_namespace() {
+    # Makes namespace of a docker container available in
+    # /var/run/netns so it can be managed with iproute2
+
+    mkdir -p /var/run/netns
+    PID="$(docker inspect -f {{"'{{.State.Pid}}'"}} "{{docker_container_name}}$1")"
+
+    if `ip netns | grep --quiet "{{docker_container_name}}$1"`; then # namespace exists
```

Comment:
if we have more than 10 namespace, it will have problem, for example when you grep database1 and there is already database11.

Reply:
fixed to grep for whole word.